test/e2e/config/azure-dev.yaml

```diff
@@ -103,7 +103,7 @@ variables:
   ETCD_VERSION_UPGRADE_TO: "3.4.3-0"
```

Comment:
@Rahul-D78 can you please update etcd and coredns versions to:

```
ETCD_VERSION_UPGRADE_TO: "3.4.13-0"
COREDNS_VERSION_UPGRADE_TO: "1.8.4"
```

Reply:
yes, sure